go/libraries/doltcore/table/untyped/xlsx/writer.go

```diff
@@ -1,117 +0,0 @@
-// Copyright 2019 Liquidata, Inc.
```

Comment:
This class never worked correctly.  It's been stripped out, and we'll need to implement.